### ISSUE TITLE
Good first issues and short proejcts in qiskit-terra

### ISSUE THREAD

**Issue by kdk** [qiskit-community/mentorship-program-fall-21#25]
# Description
Qiskit Terra maintains lists of issues believed to be either 1) good starting points into qiskit development, or 2) short, self-contained projects that will each implement a meaningful chunk of functionality.

- [Good first issues](https://github.com/Qiskit/qiskit-terra/issues?page=1&q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
- [Short projects](https://github.com/Qiskit/qiskit-terra/issues?page=1&q=is%3Aopen+is%3Aissue+label%3A%22short+project%22)

# Mentor/s
N

# Type of participant
Prior software development experience will be valuable. Experience with Qiskit or python will be helpful.

# Number of participants
N

# Deliverable
A series of commits (or resolved issues) to qiskit-terra.

**Comment by HuangJunye** [comment]
@clausia @snsunx Can you please comment in the issue so that I can assign you?

**Comment by clausia** [comment]
@HuangJunye hey!
My full name is Claudia Zendejas-Morales

**Comment by snsunx** [comment]
Hi @HuangJunye, Claudia and I will be the mentees in this project

**Comment by HuangJunye** [comment]
@clausia @snsunx Can you please upload your presentation here by the end of today? Thank you!

**Comment by snsunx** [comment]
@HuangJunye Ok got it. I'll upload my slides today. Since it's a 3-minute talk should we just have 2-3 slides?

**Comment by HuangJunye** [comment]
@snsunx Yes, 2-3 slides are sufficient. Other projects have uploaded there presentations on their project issues. You can check them out.

**Comment by snsunx** [comment]
Hi @HuangJunye, here are my slides: [#25 Good First Issues and Short Projects in qiskit-terra.pdf](https://github.com/qiskit-advocate/qamp-fall-21/files/7299715/25.Good.First.Issues.and.Short.Projects.in.qiskit-terra.pdf)

**Comment by snsunx** [comment]
I pushed three commits and had some discussions with Dr. Naoki Kanazawa on the PR. Since I made many changes which don't fit the newest design philosophy of Qiskit, I will close the PR and start again. Meanwhile I learned about how to resolve DevOps checking failures, such as how to write proper deprecation statements.

**Comment by snsunx** [comment]
Here is a visual representation of my project so far.

![Presentation1-1](https://user-images.githubusercontent.com/38364942/141421693-fd45d086-02d0-48bf-b36d-7c7f67c9b94b.png)

**Comment by HuangJunye** [comment]
@snsunx Can you please upload the final presentation slides and link your pull requests if any here? Thank you!

**Comment by snsunx** [comment]
@HuangJunye Sorry I don't have a finished pull request so I will not be able to upload it here. Thanks for organizing the program.
